pagopa/io-app
# chore: [IOBP-616] Add missing parameters to payment's `calculateFees` request payload

## Short description
This PR adds `isAllCCP` and `primaryCreditorInstitution` parameters to the payment's `calculateFees` request payload

## List of changes proposed in this pull request
- Updated pagoPA payments and wallet API definitions
- Added `isAllCCP` and `primaryCreditorInstitution` to calculateFees request payload

## How to test
In UAT environment, try to make a payment. You should be able to see the PSP list
